allennlp/training/metrics/conll_coref_scores.py

```diff
@@ -48,29 +48,38 @@ def get_gold_clusters(gold_clusters):
 
     @staticmethod
     def get_predicted_clusters(top_spans, antecedent_indices, predicted_antecedents):
-        mention_to_predicted = {}
-        predicted_clusters = []
+        mention_to_predicted: Dict[Tuple[int, int], int] = {}
```

Comment:
I read this at first as "mention to predict", not a mapping from mentions to predicted cluster ids.  I think something like `mention_clusters` or `predicted_mention_clusters` would be fine as a name here.